README.md

```diff
@@ -76,6 +76,8 @@ information on how to submit your changes to this repository.
 
 ## Sample view
 
+See ./examples for docker-compose file with Grafana if you'd like to quickly get a test environment up and running.
```

Comment:
Possible to link to the steps to get a `docker-compose` setup up and running for our non-docker savvy audience?